spec/matrix_spec.cr

```diff
@@ -403,6 +403,25 @@ describe Matrix do
     end
   end
 
+  context "trace" do
+    it "computes trace for integers" do
```

Comment:
I don't think this spec adds much value, a matrix is always backed by an array of floats